var/spack/repos/builtin/packages/arm-forge/package.py

```diff
@@ -23,6 +23,7 @@ class ArmForge(Package):
 
     if platform.machine() == "aarch64":
         version("21.1",   sha256="d6f6444eb2d47fd884d8b125f890d6a02a9d5bcfc10950af46b11d3b1e1189fd")
+        version("21.0.3", sha256="371f4e3087af329bee155ceb50b9adaf006d3b8602fb1b6bbdc710ab0f74368d")
```

Comment:
Why are we adding this version only for `aarch64`?

Reply:
Can you update other architectures too? I think it would seem odd to have a specific patch version available only for an architecture.

Reply:
At a glance I can't find the 21.0.3 source files on their website; it is a bit old.

TBH, I'd rather someone with a need for this patch version for their desired architecture open a PR themselves, as I have. Is this unreasonable (genuinely asking)?

Reply:
I'd prefer to add these for consistency:
```
$ # On x86_64
$ spack checksum arm-forge 21.0.3
[ ... ]
==> Fetching http://content.allinea.com/downloads/arm-forge-21.0.3-linux-x86_64.tar

    version('21.0.3', sha256='24708b363a8d3a82879ab4f60fe08faeb936e1e8286b324928a86a2d707071e8')

$ # On ppc64le
$ spack checksum arm-forge 21.0.3
[ ... ]
==> Fetching http://content.allinea.com/downloads/arm-forge-21.0.3-linux-ppc64le.tar

    version('21.0.3', sha256='a1dad6efdfa6bf95e5ec02f771217bf98d7d010c6bfdd5caf0796f0e75ef0fab')

```

Reply:
Neat! What a spiffy command, I didn't know you could do that.